### PR TITLE
build related fixes and 3D functionality added to kernel

### PR DESCRIPTION
**Pull request by RubendeBruin**
added 'Library\include\' to include dirs in case 'include\cgal' does not exist
removed CMakeLists.txt

new functionality:
added 3D functions

### CODE REVIEW

**Comment by RubendeBruin** [comment]
Note:
Two lines have been commented out because they refused to compile on OS-X:

//m.def("do_intersect", &do_intersect<Line_3, Segment_3>); <-- THIS ONE ERRORS ON OS-X, windows (VS) & linux work fine

//m.def("do_intersect", &do_intersect<Ray_3, Segment_3>);  <-- THIS ONE ERRORS ON OS-X, windows (VS) & linux work fine

**Comment by RubendeBruin** [comment]
@wolfv : ready for you

File: src/kernel.cpp [hunk 1]
```diff
@@ -373,11 +419,39 @@ void init_skgeom_kernel(py::module &m) {
...
         .def(py::self + Vector_3())
+        //.def(py::self + Point_3())
         .def(py::self - Point_3())
         .def(py::self - Vector_3())
         .def(py::self - CGAL::Origin())
     ;
 
+    py::class_<Line_3>(m, "Line3")
+        .def(py::init<Point_3, Point_3>())
+    	.def(py::init<Point_3, Direction_3>())
```
Comment (wolfv) [inline src/kernel.cpp:430]: hey, sorry to bother you again but the indentation still does look a little off here.

File: src/kernel.cpp [hunk 2]
```diff
@@ -346,6 +346,52 @@ void init_skgeom_kernel(py::module &m) {
...
+ 		.def(py::init<>())
+ 		//.def(py::init<CGAL::Rotation, double, double>())
+ 		//.def(py::init<CGAL::Rotation, Direction_2, double>())
+ 		.def(py::init<CGAL::Translation, Vector_3>())
+ 		.def(py::init<CGAL::Scaling, double>())
+ 		.def(py::init<double, double, double, double, double, double, double, double, double, double, double, double, double>())
+ 		.def(py::init<double, double, double, double, double, double, double, double, double, double>())
+ 		.def(py::self * Aff_transformation_3())
+        
+        .def("transform", static_cast<Point_3 (Aff_transformation_3::*)(const Point_3 &) const>(&Aff_transformation_3::transform))
```
Comment (wolfv) [inline src/kernel.cpp:371]: and here (or above) as well. If possible use only spaces.

**Comment by RubendeBruin** [comment]
Changed tabs to spaces

@wolfv : "Do you want to change your commit message slightly?" what do you mean exactly?

**Comment by wolfv** [comment]
Just that the first commit message reads: "build related: " but it's fine, merged! thanks for this PR!

**Pull request merged**